src/shared/settings.d.ts

```diff
@@ -20,7 +20,7 @@ export interface Settings {
     appBadge?: boolean;
     disableMinSize?: boolean;
     clickTrayToShowHide?: boolean;
-    customTitleBar?: boolean;
+    titleBar?: "hidden" | "shown" | "custom";
```

Comment:
the values are kinda misleading
```suggestion
    titleBar?: "inset" | "system" | "discord";
```